examples/hello-back/HelloBack.sol

```diff
@@ -0,0 +1,71 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import { IAxelarExecutable } from '@axelar-network/axelar-cgp-solidity/contracts/interfaces/IAxelarExecutable.sol';
+import { IERC20 } from '@axelar-network/axelar-cgp-solidity/contracts/interfaces/IERC20.sol';
+import { IAxelarGasService } from '@axelar-network/axelar-cgp-solidity/contracts/interfaces/IAxelarGasService.sol';
+import { StringToAddress } from 'axelar-utils-solidity/src/StringAddressUtils.sol';
+
+contract HelloBack is IAxelarExecutable {
+    using StringToAddress for string;
+
+    string public message;
+    IAxelarGasService gasReceiver;
+
+    constructor(address _gateway, address _gasReceiver) IAxelarExecutable(_gateway) {
+        gasReceiver = IAxelarGasService(_gasReceiver);
+    }
+
+    function _sayHelloBack(
+        string memory destinationChain,
+        string memory destinationAddress,
+        bytes memory payload,
+        string memory tokenSymbol,
+        uint256 gasFeeAmount,
+        address refundAddress
+    ) internal {
+        address gasToken = gateway.tokenAddresses(tokenSymbol);
+        IERC20(gasToken).approve(address(gasReceiver), gasFeeAmount);
+        gasReceiver.payGasForContractCall(
+            address(this),
+            destinationChain,
+            destinationAddress,
+            payload,
+            gasToken,
+            gasFeeAmount,
+            refundAddress
+        );
+        gateway.callContract(destinationChain, destinationAddress, payload);
+    }
+
+    function _otherCallback(
```

Comment:
change the name to something else e.g. `_transfer`?

Reply:
it's meant to be a generic callback (this case... was transfer)